tests/unit-ext/test-common.h

```diff
@@ -0,0 +1,49 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef TEST_COMMON_H
+#define TEST_COMMON_H
+
+#include "jerryscript-port.h"
+
+/*
+ * Conditions' likeliness, unlikeliness.
+ */
+#ifdef __GNUC__
+#define likely(x)       __builtin_expect(!!(x), 1)
+#define unlikely(x)     __builtin_expect(!!(x), 0)
+#else /* !__GNUC__ */
+#define likely(x)       (x)
+#define unlikely(x)     (x)
+#endif /* __GNUC__ */
+
+
+#define TEST_ASSERT(x) \
+  do \
+  { \
+    if (unlikely (!(x))) \
```

Comment:
Guiding the compiler on what is likely an what is not is important in "real" code (e.g., the engine) for performance reasons. In unit tests, not that much. We could drop unlikely, and all the preproc conditionals above.